### PR TITLE
show Appointments

### PR DESCRIPTION
**Pull request by BrainCodeJockey**

### CODE REVIEW

**Comment by esrefbozkurt** [comment]
pls check comments above

File: components/AppointmentCard.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  /* grid-template-areas:
+    "a b c"
+    "d e f"; */
+  align-content: space-between;
+  padding: 10px;
+  background-color: white;
+  border-bottom: 1.6px solid #dadada;
+`;
+
+const Name = styled.div`
```
Comment (julialoeschel) [inline components/AppointmentCard.js:31]: if you are displaying text , use a HTML Tag for Text, `p` or `span` maybe? check also for the following items.

**Review by julialoeschel — COMMENTED** [review]
great job. 

Eshis Anmerkungen sind super, ich habe nur eine weitere Bemerkung. 

weiter so! 
🐜

**Review by Hirnstromwelle — COMMENTED** [review]
Changes noted

**Pull request merged**